wgsl/index.bs

```diff
@@ -10881,11 +10891,38 @@ The rules for analyzing expressions take as argument both the expression itself
       <td class="nowrap">*CF*, *CF*
       <td class>
    <tr><td>identifier [=resolves|resolving=] to [=const-declaration=], [=override-declaration=],
-      [=let-declaration=], or non-built-in [=formal parameter=] "x"
+      [=let-declaration=], or non-built-in [=formal parameter=] of [=pointer type|non-pointer type=] "x"
       <td>*Result*
       <td>*X* is the node corresponding to "x"
       <td class="nowrap">*CF*, *Result*
       <td class="nowrap">*Result* -> {*CF*, *X*}
+   <tr><td>identifier [=resolves|resolving=] to a [=formal parameter=] of
+      [=pointer type=] in the [=address spaces/storage=], [=address spaces/workgroup=],
+      or [=address spaces/private=] [=address spaces=] with a non-read-only
+      [=access mode=] where the identifier appears as the [=root identifier=]
+      of a [=memory view=] expression, *MVE*, and the [=load rule=] is invoked
+      on *MVE* during [=type checking=]
+      <td>
+      <td>
+      <td>*CF*, [=MayBeNonUniform=]
+      <td>
+   <tr><td>identifier [=resolves|resolving=] to a [=formal parameter=] of
+      [=pointer type=] in the [=address spaces/storage=], [=address spaces/workgroup=],
+      or [=address spaces/private=] [=address spaces=] with a non-read-only
+      [=access mode=] where the identifier appears as the [=root identifier=]
+      of a [=memory view=] expression, *MVE*, and the [=load rule=] is not
+      invoked on *MVE* during [=type checking=]
+      <td>
+      <td>
+      <td>*CF*, *CF*
+      <td>
+   <tr><td>identifier [=resolves|resolving=] to a [=formal parameter=] of
```

Comment:
It took me a while to see why this completes the coverage of pointer parameters.
That's because pointers for function address space have been desugared away.

LGTM.